blog/_posts/2019-08-01-commit-of-the-month-july.md

```diff
@@ -0,0 +1,63 @@
+---
+slug: july-tilt-commit-of-the-month
+date: 2019-08-01
+author: maria
+layout: blog
+title: "Tilt Commit of the Month: July 2019"
+subtitle: "No Browser - Flag to Disable the Web UI"
+images:
+  - featuredImage.png
+image_type: "contain"
+tags:
+  - docker
+  - kubernetes
+  - microservices
+  - tilt
+  - cotm
+keywords:
+  - web-interface
+  - terminal
+  - tilt
+  - flag 
+---
+
+
+Welcome the second edition of Commit of the Month, a series of blog posts where we highlight some of the work done on Tilt this month! 
+
+This month’s Commit of the Month is the command line flag that disables the webUI:
+
+* [web: flag for opening browser](https://github.com/windmilleng/tilt/pull/1830)
+
+## What does it do?
+
+
+When you Tilt Up, usually you will see two UIs - the one that shows up on your terminal and the webUI.
+
+This one pops up on your _browser_..
+![Web UI](/assets/images/july-tilt-commit-of-the-month/webUI.png) 
+
+And this one pops up on your _terminal_.. 
+![Terminal UI](/assets/images/july-tilt-commit-of-the-month/TUI.png)
+
+Look familiar? 
+
+Well now, if you want to disable the web UI, you can now tilt up along with “--no-browser” 
+```
+tilt up --no-browser
+```
+
+## But.. why do we have two interfaces in the first place? 
+This is something we are still trying to figure out as well. At first, the terminal UI was the original Tilt - where you can see your services and their status. As we progressed through our Tilt journey, we experimented with building the web UI, which made it easier to navigate through logs and preview the services.  
```

Comment:
maybe a note on why we bothered to auto-open it? (i.e. b/c we wanted to make sure ppl knew about it!)